tests/baselines/reference/user/fp-ts.log

```diff
@@ -0,0 +1,11 @@
+Exit Code: 1
+Standard output:
+test/property-test/Option.ts(25,3): error TS2322: Type 'Arbitrary<None | Some<unknown> | Some<A>>' is not assignable to type 'Arbitrary<Option<A>>'.
```

Comment:
this is different!

Reply:
I can't think of an obvious candidate for this one, tbh.